src/CorsService.php

```diff
@@ -414,7 +470,26 @@ protected function isAllOriginsAllowed()
     {
         return in_array('*', $this->allowOrigins);
     }
+	
+	
+    /**
+      * @return array
+      */
+    protected function getWildcardOrigins()
+    {
+        // We'll search for an *. in all the allowed origins
+        return preg_grep('#\*\.#', $this->allowOrigins);
+    }
 
+    /**
+      * @return bool
+     */
+    protected function hasWildcardOrigins()
+    {
+        $matchedOrigins = $this->getWildcardOrigins();
+        
+        return (count($matchedOrigins) > 0);
```

Comment:
No need for parenthesis, it’s one expression.

Reply:
True, it fact its only there to help me read and evaluate them, I get it as an implicit bool cast.